frontend/src/components/v1/InlineAlertTray.css

```
@@ -0,0 +1,127 @@
+  gap: 0.75rem;
+  padding: 0.75rem 1rem;
+  font-family: var(--font-family, inherit);
+  font-size: 0.875rem;
+  line-height: 1.4;
+  border-radius: 6px;
+  animation: slideDownFadeIn 0.3s cubic-bezier(0.16, 1, 0.3, 1);
+  box-sizing: border-box;
+  width: 100%;
+}
+
+@keyframes slideDownFadeIn {
```

Stylelint (17.12.0)

[error] Expected keyframe name "slideDownFadeIn" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)